i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-commercial-extensions.md

```diff
@@ -15,6 +15,51 @@ Pour faire des demandes d'évolutions ou reporter des bugs sur les extensions co
+
+- [API] Fixed an issue with create element in view that could break default icon.
+- [Authentication] Fixed an issue with authentication when several sessions are active.
+- [CEIP] Added more details for MAP engine telemetry.
+- [Editor] Fixed an issue in shapes edition modal that caused scrollbar to be reset when browsing images.
+- [Editor] Fixed an issue that prevented users from drawing rectangles and ellipses.
+- [Editor] Fixed an issue where users could not edit pie chart properties.
+- [Editor] Fixed an issue with Host and Service icons configuration that were not handled properly.
+- [Editor] Fixed an issue with insert an object in a link prevented the map from being saved.
+- [Editor] Fixed an issue with link properties that prevented the link from being visible in the editor.
+- [Editor] Fixed an issue with links in the editor that prevented the user from saving a view.
+- [Editor] Fixed an issue with map sharing.
+- [Install] Fixed an issue preventing the installation of **centreon-map-engine** on the same machine as **centreon-map-server**.
+- [Install] Fixed the **diagnostic.sh** script to include storage event from Broker output.
+- [Monitoring] Fixed an issue where MAP was not supporting performance data returned by Cisco ASA plugin.
+- [Server] Fixed an issue that caused server to crash on corrupted metrics.
+- [Server] Fixed an issue that made new hosts and services unavailable until restart.
+- [Server] Fixed an issue that prevented the migration between MAP Legacy and MAP.
+- [Server] Fixed an issue with migration and images that could cause migration process to stop.
+- [Server] Fixed an issue with the server startup error caused by platform without virtual hosts.
+- [Viewer] Fixed an issue that could cause a blank page to be displayed when user clicked on host name link inside a map.
+- [Viewer] Fixed an issue that prevented the tooltip for the "old perfdata" plugin from being displayed in the graph.
+- [Viewer] Fixed an issue when custom URL was set and prevented some old media in maps from being displayed correctly.
+- [Viewer] Fixed an issue when dark theme was not applied on tooltips in Geo views or Standard views.
+- [Viewer] Fixed an issue with colors for a "Text" type element that was ignored by the viewer.
+- [Viewer] Fixed the background layout not fitting large map.
+- [Viewer] Fixed view edition with some links having corrupted bend points.
+
+#### Enhancements
+
+- [API] Added documentation and sanitization of the API.
+- [API] Improved API response when server is starting up.
+- [API] Removed deprecated REST endpoints.
+- [Editor] Added resource parent name in graph.
+- [Editor] Added spacing to the title inside the MAP configuration modal.
+- [Editor] Added the display of supported shapes only at the double-click on the editor.
+- [Editor] Aligned "Submit button" inside the MAP configuration modal.
+- [Editor] Managed view background color.
+- [Viewer] Made viewer and editor rendering more consistent.
+
```

Comment:
missing 12115
Fixed an issue that caused the unit not matching the axis on graph widget.